src/admin-views/aggregator/settings.php

```diff
@@ -364,6 +364,38 @@
 
 $internal = apply_filters( 'tribe_aggregator_fields', $internal );
 
+// fix this shit
```

Comment:
Could remove this line before we merge (or change to a more formal @todo)? :wink: